marketplace_builder/views/pages/tutorials/custom-model-types/using-geojson-format-geolocation.liquid

```diff
@@ -148,14 +142,34 @@ The HTML field snippet for geometry JSON is just a textarea:
   <textarea required="true" id="location" type="text" class="form-control" name="{{ form_builder.fields.properties.location.name }}">
       {{- context.params.form.properties_attributes.location }}
   </textarea>
-</div> 
+</div>
 ```
 
 {% capture content %}
 This example is for presentation only. In a real scenario, no one would provide a raw JSON as a location address — there would be a JavaScript function using external geolocation services for getting coordinates and then building the required GeometryJSON structure.
 {% endcapture %}
 {% include 'alert/note', content: content %}
 
+### Step 3: Prepare GraphQL query
+
+Create the GraphQL query `graph_queries/search.graphql`:
+
+```graphql
+query search($point: GeometryJSON, $distance: Int) {
+  companies: models(
+    per_page: 20
+    filter: {
+      properties: [
+        { name: "location", distance_sphere: {center: $point, distance_in_km: $distance}}
+      ]
+    }) {
+    results {
+      properties
+    }
+  }
+}
+```
+
```

Comment:
This will search across all the models right?  Can we add some model name? People tend to copy paste those example and thy will probably search across all models too...

Reply:
@lemingos thx. updated